tests/bug-1005.sil

```diff
@@ -0,0 +1,4 @@
+\begin{document}
```

Comment:
All tests (unless specifically testing paper size settings or having other good reasons to change it) should use the small paper sizes. This makes graphic review when needed much easier.

Also always include `\nofolios` when not testing the folios themselves because it simplifies the test output and the number of things unrelated to the test that might change. Likewise `\noindent` or `\neverindent` is quite useful to normalize tests where those settings aren't relevant.

```suggestion
\begin[papersize=a6]{document}
\nofolios
\neverindent
```